### PR TITLE
Command to automatically clear messages in channel

### LINKED ISSUE #39: Automatic message expiration in sensitive channels
Certain channels like `#vent-and-serious` and `#society` may contain personal information about other people.

The bot should have a configurable feature that automatically deletes messages in these channels after a certain period of time.

### PR DESCRIPTION
**Pull request by thederekkaplan**
Closes #39

### CODE REVIEW

File: src/Api.ts [hunk 1]
```diff
@@ -306,6 +320,37 @@ class Api {
...
+   * Gets messages from the specified channel.
+   *
+   * @param limit The max number of messages to return (1-100)
+   * @param channelId The ID of the channel
+   * @param filter Object to filter messages
+   */
+  getMessages(
+    channelId: string,
+    limit = 50,
+    filter: FilterType = undefined,
```
Comment (j-651) [inline src/Api.ts:333]: Is it necessary to specify "undefined" here versus just `filter: FilterType,`?

File: src/jobHandlers/autoclear.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  guildId: string;
+};
+
+const autoclear: JobHandler<JobType.AUTOCLEAR> = async ctx => {
+  const { guildId } = ctx.data;
+
+  const today = dayjs();
+
+  const channels = await (await ctx.repos.autoclear.scan(guildId)).toArray();
+  console.log(channels);
```
Comment (jaysella) [inline src/jobHandlers/autoclear.ts]: Let's remove this before merging

**Review by jaysella — APPROVED** [review]
This looks good!

File: src/Api.ts (hunk 1 above)
Comment (thederekkaplan) [inline src/Api.ts:333]: Specifying `= undefined` gives it a default value so it's not a mandatory parameter.

File: src/jobHandlers/autoclear.ts (hunk 2 above)
Comment (thederekkaplan) [reply src/jobHandlers/autoclear.ts]: oh lmao thank you for catching this

File: src/Api.ts (hunk 1 above)
Comment (jaysella) [reply src/Api.ts:333]: Yes yes yes, my bad :)

**Pull request merged**